### PR TITLE
Fix/screen reader improvements

### LINKED ISSUE #205: Collection view based rows aren't scrollable using VoiceOver
## Expected Behaviour

The user should be able to use 3-finger swipe or swipe to navigate through items in a horizontally scrollable list item using VoiceOver.

## Current Behaviour

Only the first item in collection view based list items is reachable

## Steps to Reproduce (for bugs)

1. Create a content page with a spotlight or app collection e.t.c.
2. Enable VoiceOver on the test device
3. Visit the created page
4. Witness that you can't scroll between items in the spotlight/collection

## Context

We are trying to make Storm ADA compliant. This is required to do so!

### PR DESCRIPTION
**Pull request by simonmitchell**
## Description

Improves screen reader / VoiceOver compatibility when using our apps. Makes sure horizontally scrolling collection items are scrollable and fully accessible

## Related Issue

Fixes #205 

## Motivation and Context

Solves storm not being fully ADA compliant!

## How Has This Been Tested?

Has been tested by using this branch locally for ARC FA

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-github "3sidedcube/ThunderTable" == 1.2.2
+github "3sidedcube/ThunderTable" "feature/screen_reader"
```
Comment (simonmitchell) [inline Cartfile]: @ryanbourneuk will change this to 1.2.3 when has been released!

Comment (ryanbourneuk) [reply Cartfile]: 👌

File: ThunderCloud/AccessibleCollectionView.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+/// and returns the correct value from `accessibilityElementCount()`
+public class AccessibleCollectionView: UICollectionView {
+
+    override open func accessibilityElementCount() -> Int {
+        
+        guard let dataSource = dataSource else {
+            return 0
+        }
+        
+        let numberOfSections = dataSource.numberOfSections?(in: self) ?? 1
```
Comment (ryanbourneuk) [inline ThunderCloud/AccessibleCollectionView.swift]: Why is this defaulting to 1 if there's no sections? :)

Comment (simonmitchell) [reply ThunderCloud/AccessibleCollectionView.swift]: Not sure... may have been a copy pasta ;) will adjust!

**Pull request merged**